core/domain/stats_jobs_one_off.py

```diff
@@ -132,53 +140,113 @@ def reduce(exp_id, stringified_values):
         exploration = exp_services.get_exploration_by_id(exp_id)
```

Comment:
Might want to check the case where an exploration is deleted.

Reply:
Done.